### DISCUSSION TITLE
Energy bar (stamina)

### DISCUSSION THREAD

**Discussion by diasfigor** [sunflower-land/sunflower-land#1533] [Ideas]
Good afternoon. A good idea would be to put a countdown timer to know how much time is left for the energy bar (stamina) to be full. I think it would be very helpful so that we don't waste time in planting and harvesting.
Thank you SunFlower team you are the best.

**Comment by InuBakaBo** [comment]
Based on what I've seen on testnet, this is already in place.

**Reply by diasfigor** [reply]
So I don't know which version of tesnet I'm using, but this countdown doesn't have it, what it has is showing the stamina going up but the time it takes to fill everything.
